vtexdocs/helpcenter
# feat: add redirects

#### What is the purpose of this pull request?



adding redirects with a netlify.toml file generated by https://github.com/vtexdocs/help-center-content/pull/52

#### What problem is this solving?

[EDU-13140](https://vtex-dev.atlassian.net/browse/EDU-13140)

#### How should this be manually tested?

1. go to the deploy preview.
2. access paths listed as 'from' in the netlify.toml file and check if you are redirected as expected.

#### Screenshots or example usage

#### Types of changes
